fms/models/__init__.py

```diff
@@ -93,14 +97,20 @@ def __maybe_infer_model_variant(
                     f"ignoring the following parameters as a pretrained model with an inferred configuration is being loaded: {list(kwargs.keys())}"
                 )
 
+        if architecture in ("hf_pretrained", "hf_configured"):
+            model_path_or_variant = variant
+        elif is_hf_inferred:
+            model_path_or_variant = model_path
```

Comment:
currently `variant` is not optional, would this now require making the `variant` optional?

Reply:
done